deepwell/src/services/alias/service.rs

```diff
@@ -112,17 +109,25 @@ impl AliasService {
                     error!(
                         "User with conflicting slug '{slug}' already exists, cannot create alias",
                     );
-
                     return Err(Error::UserExists);
                 }
 
-                if slug.len() < ctx.config().minimum_name_bytes {
+                let config = ctx.config();
+                if slug.len() < config.minimum_name_bytes {
                     error!(
-                        "User's name is not long enough ({} < {})",
+                        "User's name is not long enough ({} < {} bytes)",
                         slug.len(),
                         ctx.config().minimum_name_bytes,
                     );
+                    return Err(Error::UserNameTooShort);
+                }
 
+                if slug.chars().count() < config.minimum_name_chars {
+                    error!(
+                        "User's name is not long enough ({} < {} chars)",
+                        slug.len(),
+                        ctx.config().minimum_name_bytes,
```

Comment:
Also change this to show min chars instead of min bytes?

Reply:
I guess I'm not on my game today. Fixed.